### PR TITLE
Add editorialized release notes for version 19.2

### PR DESCRIPTION
**Pull request by mokagio**
PR with no beta going with it so we can get these changes into `trunk` and out for translation ASAP.

The diff also contains the work from #17891, but that touches the CI automation only, not the app's codebase itself.

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] `AppStoreStrings.po(t)` changed with editorialized release notes

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
-* [*] Block editor: Replacing the media for an image set as featured prompts to update the featured image [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930]
-* [***] Block editor: Font size and line-height support for text-based blocks used in block-based themes [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4519]
-* [**] Some of the screens of the app has a new, fresh and more modern visual, including the initial one: My Site. [#17812]
-* [**] Notifications: added a button to mark all notifications in the selected filter as read. [#17840]
-* [**] People: you can now manage Email Followers on the People section! [#17854]
-* [*] Stats: fix navigation between Stats tab. [#17856]
-* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17888]
-* [*] Stats: Users can now add a new Insights card from the navigation bar. [#17867]
+With this latest version of Jetpack, we made some design changes to help make your user experience even better.
```
Comment (ParaskP7) [inline WordPress/Jetpack/Resources/release_notes.txt:1]: Good job spotting the `WordPress` vs `Jetpack` typo from the editorial review! 🥇

**Review by ParaskP7 — APPROVED** [review]
Ship it! 🟢

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 1 above)
Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt:1]: Thanks 😄

**Pull request merged**